packages/@expo/fingerprint/src/Fingerprint.types.ts

```diff
@@ -39,9 +39,21 @@ export interface Options {
   /**
    * Excludes directories from hashing. This supported pattern is as `glob()`.
    * Default is `['android/build', 'android/app/build', 'android/app/.cxx', 'ios/Pods']`.
+   * @deprecated Use `ignores` instead.
    */
   dirExcludes?: string[];
 
+  /**
+   * Ignore files and directories from hashing. This supported pattern is as `glob()`.
+   *
+   * Please note that the pattern matching is slightly different from gitignore. For example, we don't support partial matching where `build` does not match `android/build`. You should use `'**' + '/build'` instead.
+   * @see [minimatch implementations](https://github.com/isaacs/minimatch#comparisons-to-other-fnmatchglob-implementations) for more reference.
+   *
+   * Besides this `ignores`, fingerprint comes with implicit default ignores defined in `Options.DEFAULT_IGNORES`.
+   * If you want to override the default ignores, use `!` prefix.
+   */
+  ignores?: string[];
```

Comment:
```suggestion
  ignorePaths?: string[];
```
in order of preference: `ignorePaths` > `ignoreDirs` > `ignores`